CorsixTH/Src/th_gfx.cpp

```diff
@@ -32,49 +32,49 @@ SOFTWARE.
 #include <cassert>
 
 /** Data retrieval class, simulating sequential access to the data, keeping track of available length. */
-class Input
+class memory_reader
 {
 public:
-    Input(const uint8_t *pData, size_t iLength)
+    memory_reader(const uint8_t *pData, size_t iLength)
     {
-        m_pData = pData;
-        m_iLength = iLength;
+        data = pData;
+        remaining_bytes = iLength;
     }
 
-    const uint8_t *m_pData; ///< Pointer to the remaining data.
-    size_t m_iLength;       ///< Remaining number of bytes.
+    const uint8_t *data; ///< Pointer to the remaining data.
+    size_t remaining_bytes;       ///< Remaining number of bytes.
```

Comment:
Documentation comment alignment is messed up, that needs to be addressed at some point.
While I like vertical alignment, if we want to change convention, now is likely a good time.

Reply:
Formatting is my next PR in this series